js/pages/estimation/cohort-comparison-manager.html

```diff
@@ -1,622 +1,620 @@
-<div class="wrapperTitle" data-bind="visible:loading">
-  <i class="fa fa-circle-o-notch fa-spin"></i> Loading...
-</div>
-
-<div data-bind="if:!loading() && cohortComparison()">
-  <div class="wrapperTitle">
-    <i class="fa fa-balance-scale"></i> Population Level Effect Estimation
-  </div>
-
-  <div class="asset-heading">
-    <input type="text" data-bind="textInput: cohortComparison().name, css: { emptyInput: !(cohortComparison().name() && (cohortComparison().name().length > 0)) }" />
-    <div class="btn-group" role="group">
-      <button class="btn btn-sm btn-success" title="Save" data-bind="click: save, enable:canSave"><i class="fa fa-save"></i></button>
-      <button class="btn btn-sm btn-primary" title="Close" data-bind="click: close"><i class="fa fa-times"></i></button>
-      <button class="btn btn-sm btn-danger" title="Delete" data-bind="click: $component.delete, enable: canDelete()"><i class="fa fa-trash-o"></i></button>
-    </div>
-  </div>
-
-  <ul class="nav nav-tabs">
-    <li role="presentation" data-bind="css: { active: $component.tabMode() == 'specification' }, click: function() { $component.tabMode('specification') };"><a>Specification</a>
-    </li>
-    <li role="presentation" data-bind="visible: config.cohortComparisonResultsEnabled, css: { active: $component.tabMode() == 'results' }, click: function() { $component.tabMode('results') };"><a>Results</a>
-    </li>
-    <li role="presentation" data-bind="css: { active: $component.tabMode() == 'export' }, click: function() { $component.tabMode('export') };"><a>Utilities</a>
-    </li>
-  </ul>
-
-  <div class="tab-content">
-    <div class="paddedWrapper text-center error" data-bind="visible:$component.loadingExecutionFailure">
-      <div>
-        <i class="fa fa-exclamation-circle"></i>
-      </div>
-      <div>error loading selected results</div>
-    </div>
-
-    <div role="tabpanel" data-bind="css: { active: $component.tabMode() == 'results' }" class="tab-pane">
-      <div class="padded-wrapper" data-bind="visible:$component.resultsMode() == 'sources'">
-        <div class="heading">Available Data Sources</div>
-        <div data-bind="foreach:$component.sources" class="results-tab-pane">
-          <div class="row">
-            <div class="col-sm-8" data-bind="text:sourceName"></div>
-            <div class="col-sm-4 text-right">
-              <span class="btn-group">
-								<button data-bind="css:{disabled: $component.config.api.isExecutionEngineAvailable()==false}, click:function() {$component.executeCohortComparison(sourceKey)}" class="btn btn-sm btn-primary"><i class="fa fa-play"></i> Generate</button>

```

Comment:
Getting an error on this line due to the fact that config.api.isExecutionEngineAvaiable() is not a function. I see that this is set in the /js/Application.js but not sure why the observable is no longer available at this point?